docs/content/4.sdk/2.getting-started/2.middleware-module.md

```diff
@@ -242,6 +251,40 @@ const product = await sdk.commerce.getProduct(
 );
 ```
 
+### Log requests and responses
+
+You can enable logging of the requests and responses by setting the `logger` option to `true` in the `middlewareModule` function.
+
+```ts
+export const sdk = initSDK({
+  commerce: buildModule(middlewareModule<UnifiedEndpoints>, {
+    apiUrl: "http://localhost:8181/commerce",
+    logger: true,
+  }),
+});
+```
+
+You can also enable logging of the requests and responses in all middleware modules by setting the environment variable `ALOKAI_SDK_DEBUG` to `true`.
```

Comment:
It's good to write what has a priority - boolean in config or an ENV?

Reply:
done [57744f3](https://github.com/vuestorefront/vue-storefront/pull/7227/commits/57744f35462c42ebe1fca319fddad81b3621d329)